utils/interactive_dataset_creator.py

```diff
@@ -25,14 +27,20 @@ def receive_operation():
 def combine(dataset_dfs):
     return pd.concat(dataset_dfs)
 
-def binarize(dataframe, positive_class):
-    dataframe.loc[dataframe['label_type'] != positive_class, 'label_type'] = 0
-    dataframe.loc[dataframe['label_type'] == positive_class, 'label_type'] = 1
+def binarize(dataframe, positive_class, is_label_set=True):
+    if is_label_set:
+        dataframe.loc[dataframe['label_set'].str.contains(str(positive_class)) == False, 'label_type'] = 0
```

Comment:
Could we try one of the solutions in this stack overflow post: [https://stackoverflow.com/questions/23111990/pandas-dataframe-stored-list-as-string-how-to-convert-back-to-list](https://stackoverflow.com/questions/23111990/pandas-dataframe-stored-list-as-string-how-to-convert-back-to-list)?

Or have we already tried these strats?